src/main/scala/nl.knaw.dans.easy.multideposit/parser/ParserValidation.scala

```diff
@@ -91,4 +94,18 @@ trait ParserValidation extends DebugEnhancedLogging {
       case accs => ParseError(deposit.row, s"Multiple accessibility levels found for A/V files: ${ accs.mkString("{", ", ", "}") }").toInvalid
     }
   }
+
+  def checkAllAVFileNamesWithoutSpaces(deposit: Deposit): Validated[Unit] = {
+    deposit.files.collect { case fmd: AVFileMetadata => fmd.filepath.name }
+      .toList
+      .traverse(name => noSpaces(deposit.row, name))
+      .map(_ => ())
+  }
+
+  private def noSpaces(row: Int, s: String): Validated[String] = {
```

Comment:
If you curry `noSpaces`, it looks even better 😉

```suggestion
      .traverse(noSpaces(deposit.row))
      .map(_ => ())
  }

  private def noSpaces(row: Int)(s: String): Validated[String] = {
```

Reply:
good idea!